cli.py

```diff
@@ -22,6 +29,13 @@ class CovidExcelUtils:
     def __init__(self, file_path, output):
         self.__file_path = file_path
         self.__output = output
+        self.ena_submission_files = {}
+
+        self.bio_studies_service = BioStudies(
+            args['biostudies_url'],
+            os.environ['BIOSTUDIES_USERNAME'],
+            os.environ['BIOSTUDIES_PASSWORD']
+        )
```

Comment:
this breaks the cli if the environment variables aren't present.
The CovidExcelUtilsClass shouldn't use any environment variables directly, the main method below should convert from the os variables to keep the class implementation clean.
```
➜  cli.py examples/12_v3_raw_reads.xlsx --output json
Traceback (most recent call last):
  File "/Users/astaffer/.pyenv/versions/3.8.3/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/astaffer/.pyenv/versions/3.8.3/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/astaffer/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/__main__.py", line 45, in <module>
    cli.main()
  File "/Users/astaffer/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/../debugpy/server/cli.py", line 444, in main
    run()
  File "/Users/astaffer/.vscode/extensions/ms-python.python-2021.1.502429796/pythonFiles/lib/python/debugpy/../debugpy/server/cli.py", line 285, in run_file
    runpy.run_path(target_as_str, run_name=compat.force_str("__main__"))
  File "/Users/astaffer/.pyenv/versions/3.8.3/lib/python3.8/runpy.py", line 265, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "/Users/astaffer/.pyenv/versions/3.8.3/lib/python3.8/runpy.py", line 97, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/Users/astaffer/.pyenv/versions/3.8.3/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/astaffer/Repos/covid-excel-utils/cli.py", line 208, in <module>
    with closing(CovidExcelUtils(args['file_path'], args['output'])) as excel_utils:
  File "/Users/astaffer/Repos/covid-excel-utils/cli.py", line 36, in __init__
    os.environ['BIOSTUDIES_USERNAME'],
  File "/Users/astaffer/.pyenv/versions/3.8.3/lib/python3.8/os.py", line 675, in __getitem__
    raise KeyError(key) from None
KeyError: 'BIOSTUDIES_USERNAME'

```